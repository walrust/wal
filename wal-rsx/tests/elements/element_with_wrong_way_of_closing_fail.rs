use wal_rsx::rsx;

fn main() {
    rsx! { <area></area> };
    rsx! { <base></base> };
    rsx! { <br></br> };
    rsx! { <col></col> };
    rsx! { <embed></embed> };
    rsx! { <hr></hr> };
    rsx! { <img></img> };
    rsx! { <input></input> };
    rsx! { <link></link> };
    rsx! { <meta></meta> };
    rsx! { <param></param> };
    rsx! { <source></source> };
    rsx! { <track></track> };
    rsx! { <wbr></wbr> };
}
