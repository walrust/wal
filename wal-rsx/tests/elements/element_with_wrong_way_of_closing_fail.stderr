error: The `<area>` tag is a void element. Void elements should be self closing (they can not have children). (hint: try `<area/>`)
 --> tests/elements/element_with_wrong_way_of_closing_fail.rs:4:12
  |
4 |     rsx! { <area></area> };
  |            ^^^^^^

error: The `<base>` tag is a void element. Void elements should be self closing (they can not have children). (hint: try `<base/>`)
 --> tests/elements/element_with_wrong_way_of_closing_fail.rs:5:12
  |
5 |     rsx! { <base></base> };
  |            ^^^^^^

error: The `<br>` tag is a void element. Void elements should be self closing (they can not have children). (hint: try `<br/>`)
 --> tests/elements/element_with_wrong_way_of_closing_fail.rs:6:12
  |
6 |     rsx! { <br></br> };
  |            ^^^^

error: The `<col>` tag is a void element. Void elements should be self closing (they can not have children). (hint: try `<col/>`)
 --> tests/elements/element_with_wrong_way_of_closing_fail.rs:7:12
  |
7 |     rsx! { <col></col> };
  |            ^^^^^

error: The `<embed>` tag is a void element. Void elements should be self closing (they can not have children). (hint: try `<embed/>`)
 --> tests/elements/element_with_wrong_way_of_closing_fail.rs:8:12
  |
8 |     rsx! { <embed></embed> };
  |            ^^^^^^^

error: The `<hr>` tag is a void element. Void elements should be self closing (they can not have children). (hint: try `<hr/>`)
 --> tests/elements/element_with_wrong_way_of_closing_fail.rs:9:12
  |
9 |     rsx! { <hr></hr> };
  |            ^^^^

error: The `<img>` tag is a void element. Void elements should be self closing (they can not have children). (hint: try `<img/>`)
  --> tests/elements/element_with_wrong_way_of_closing_fail.rs:10:12
   |
10 |     rsx! { <img></img> };
   |            ^^^^^

error: The `<input>` tag is a void element. Void elements should be self closing (they can not have children). (hint: try `<input/>`)
  --> tests/elements/element_with_wrong_way_of_closing_fail.rs:11:12
   |
11 |     rsx! { <input></input> };
   |            ^^^^^^^

error: The `<link>` tag is a void element. Void elements should be self closing (they can not have children). (hint: try `<link/>`)
  --> tests/elements/element_with_wrong_way_of_closing_fail.rs:12:12
   |
12 |     rsx! { <link></link> };
   |            ^^^^^^

error: The `<meta>` tag is a void element. Void elements should be self closing (they can not have children). (hint: try `<meta/>`)
  --> tests/elements/element_with_wrong_way_of_closing_fail.rs:13:12
   |
13 |     rsx! { <meta></meta> };
   |            ^^^^^^

error: The `<param>` tag is a void element. Void elements should be self closing (they can not have children). (hint: try `<param/>`)
  --> tests/elements/element_with_wrong_way_of_closing_fail.rs:14:12
   |
14 |     rsx! { <param></param> };
   |            ^^^^^^^

error: The `<source>` tag is a void element. Void elements should be self closing (they can not have children). (hint: try `<source/>`)
  --> tests/elements/element_with_wrong_way_of_closing_fail.rs:15:12
   |
15 |     rsx! { <source></source> };
   |            ^^^^^^^^

error: The `<track>` tag is a void element. Void elements should be self closing (they can not have children). (hint: try `<track/>`)
  --> tests/elements/element_with_wrong_way_of_closing_fail.rs:16:12
   |
16 |     rsx! { <track></track> };
   |            ^^^^^^^

error: The `<wbr>` tag is a void element. Void elements should be self closing (they can not have children). (hint: try `<wbr/>`)
  --> tests/elements/element_with_wrong_way_of_closing_fail.rs:17:12
   |
17 |     rsx! { <wbr></wbr> };
   |            ^^^^^
